src/styles/document-style.css

```diff
@@ -248,21 +287,14 @@ p {
 }
 
 @media only screen and (max-width: 400px) {
-	:root {
+	html {
 		font-size: 60%;
```

Comment:
On above media queries you were decreasing the font size on decreasing the screen size , then why you have increased for this screen size ?

Reply:
I will fix that. Thanks for review